### PR TITLE
dts: bindings: Fix KSZ8081 property names

### PR DESCRIPTION
**Pull request by decsny**
Fix KSZ8081 binding properties:
- reset-gpios and interrupt-gpios are generally standard properties and therefore should not be using a special name
- mc, is not the correct vendor prefix for microchip

### CODE REVIEW

**Review by stanislav-poboril — APPROVED** [review]

**Review by lmajewski — APPROVED** [review]

**Review by dleach02 — DISMISSED** [review]

**Review by pdgendt — DISMISSED** [review]

File: dts/bindings/ethernet/microchip,ksz8081.yaml [hunk 1]
```diff
@@ -8,15 +8,13 @@ compatible: "microchip,ksz8081"
 include: ethernet-phy.yaml
 
 properties:
-  mc,reset-gpio:
+  reset-gpios:
     type: phandle-array
-    specifier-space: gpio
     description: GPIO connected to PHY reset signal pin. Reset is active low.
-  mc,interrupt-gpio:
+  interrupt-gpios:
```
Comment (stanislav-poboril) [inline dts/bindings/ethernet/microchip,ksz8081.yaml:14]: Looking at `dts/bindings` folder, `int-gpios` name is used often, `interrupt-gpios` only once.

**Review by stanislav-poboril — CHANGES_REQUESTED** [review]
Looking at dts/bindings folder, int-gpios name is used often, interrupt-gpios only once.

File: dts/bindings/ethernet/microchip,ksz8081.yaml (hunk 1 above)
Comment (decsny) [reply dts/bindings/ethernet/microchip,ksz8081.yaml:14]: true, I'll change it to this

**Review by pdgendt — APPROVED** [review]

**Review by stanislav-poboril — APPROVED** [review]

**Review by stanislav-poboril — APPROVED** [review]

**Review by pdgendt — APPROVED** [review]

**Review by EmilioCBen — APPROVED** [review]

**Comment by decsny** [comment]
reassigned to myself because I introduced this binding and driver and all these usages of it

**Review by MaureenHelm — COMMENTED** [review]
Need to mention in the migration guide

**Comment by decsny** [comment]
> Need to mention in the migration guide

I was planning to go through the commits since 3.6 to Ethernet and make sure the release notes and migration guide are done before the release , so I'll include it then

**Pull request merged**